stories/topics/ref-aap-aws-uninstall.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref-aws-uninstall"]
+
+= Uninstall
+
+The following document guides how to uninstall {PlatformNameShort} from {Market}. If extension nodes were deployed, a prerequisite step must be taken to remove the extension nodes before uninstalling the stack.
```

Comment:
This section describes how to uninstall {PlatformNameShort} from {Market}. If you deployed extension nodes, you must take the following steps to remove the extension nodes before uninstalling the stack.

Reply:
Fixed!